<ui version="4.0" >
 <class>LineEditorItem</class>
 <widget class="QWidget" name="LineEditorItem" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>29</height>
   </rect>
  </property>
  <property name="baseSize" >
   <size>
    <width>100</width>
    <height>20</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>lineEditorItem</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>3</number>
   </property>
   <property name="margin" >
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="nameLabel" >
     <property name="minimumSize" >
      <size>
       <width>240</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>240</width>
       <height>30</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEdit" >
       <property name="minimumSize" >
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>176</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton" >
     <property name="text" >
      <string>auto apply</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="QtAutoGui.qrc" />
 </resources>
 <connections/>
</ui>
